.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+<!-- 
```

Comment:
Do we need the `<!-- ` and `-->`?

Reply:
I can remove them, but then the template would show up in the actual PR body.

Reply:
No problem. I don't really know that is the standard behavior for things like that. I guess that we should leave it commented out.

Reply:
I was mostly going off of https://raw.githubusercontent.com/conda/conda/master/.github/ISSUE_TEMPLATE.md

Reply:
That is fair, although maybe out of scope for this PR?